test/ctr.bats

```diff
@@ -450,6 +450,66 @@ function teardown() {
 	stop_ocid
 }
 
+@test "ctr execsync exit code" {
+	start_ocid
+	run ocic pod run --config "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+	run ocic ctr create --config "$TESTDATA"/container_redis.json --pod "$pod_id"
+	echo "$output"
+	[ "$status" -eq 0 ]
+	ctr_id="$output"
+	run ocic ctr start --id "$ctr_id"
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ocic ctr execsync --id "$ctr_id" exit 42
```

Comment:
Yeah, I'll switch it to use `false`. I'm a bit disappointed that we can't test some various `ExitCode`s but `false` should be good enough.